dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -14,7 +14,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 {
     inline const size_t c_maxAppNotificationPayload{ 5120 };
     inline const uint8_t c_maxTextElements{ 3 };
-    inline const uint8_t c_maxInputElements{ 5 };
+    inline const uint8_t c_maxButtonElements{ 5 };
```

Comment:
NIT: `constexpr`

Reply:
Addressed in https://github.com/microsoft/WindowsAppSDK/pull/2801